jimp.d.ts

```diff
@@ -1,5 +1,28 @@
 declare namespace Jimp {
-    type ImageCallback = (err: Error | null, image: Jimp) => any;
+    type GenericCallback<T, U = any, TThis = any> = (
+        this: TThis,

```

Comment:
is `TThis` a mistake?

Reply:
It's not a mistake, it's a generic argument that specifies the value of `this` in the callback function